tests/unit/models/keras/test_sampler.py

```diff
@@ -100,3 +103,77 @@ def test_ensemble_trajectory_sampler_resample_provides_new_samples_without_retra
     npt.assert_array_less(1e-4, tf.abs(evals_1 - evals_2))
     npt.assert_array_less(1e-4, tf.abs(evals_2 - evals_3))
     npt.assert_array_less(1e-4, tf.abs(evals_1 - evals_3))
+
+
+@pytest.mark.parametrize("num_evals", [10, 20])
+def test_dropout_trajectory_sampler_returns_trajectory_function_with_correctly_shaped_output(
```

Comment:
Should we mark these tests with pytest.mark.mcdropout as well?

Reply:
Yes!